src/main/java/uk/gov/companieshouse/pscstatementdataapi/controller/PscStatementController.java

```diff
@@ -55,10 +57,10 @@ public ResponseEntity<StatementList> searchPscStatementsList(
             @RequestParam(value = "start_index", required = false, defaultValue = "0") final Integer startIndex) throws JsonProcessingException, ResourceNotFoundException {
         logger.info(String.format("Retrieving psc statement list data for company number %s, start index %d, items per page %d", company_number,
                 startIndex,
-                itemsPerPage));
+                Math.min(itemsPerPage, 100)));
```

Comment:
Is it worth just re-setting itemsPerPage here to save on the same min calculation twice?